### PR TITLE
Escalation support + error code adjustment

### PR DESCRIPTION
**Pull request by barmac**
image

image

Related to https://github.com/camunda/camunda-modeler/issues/3318

### CODE REVIEW

**Comment by barmac** [comment]
I missed one part: https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/79

File: lib/utils/error-messages.js [hunk 1]
```diff
@@ -463,4 +471,22 @@ function getSupportedMessage(prefix, executionPlatform, executionPlatformVersion
...
+  } = report;
+
+  const {
+    node,
+    parentNode,
+    property
+  } = data;
+
+  if (is(node, 'bpmn:Escalation') && property === 'escalationCode' && is(parentNode, 'bpmn:CatchEvent')) {
+    return 'Escalation code must be a static value in a catch event';
```
Comment (nikku) [inline lib/utils/error-messages.js:488]:
```suggestion
    return 'Escalation code in catch event must be a static value';
```

File: lib/utils/error-messages.js [hunk 2]
```diff
@@ -463,4 +471,22 @@ function getSupportedMessage(prefix, executionPlatform, executionPlatformVersion
...
+  } else if (is(node, 'bpmn:Error') && property === 'errorCode' && is(parentNode, 'bpmn:CatchEvent')) {
+    return 'Error code must be a static value in a catch event';
```
Comment (nikku) [inline lib/utils/error-messages.js:490]:
```suggestion
    return 'Error code used in catch event must be a static value';
```

Comment (barmac) [reply lib/utils/error-messages.js]: Adjusted.

**Comment by barmac** [comment]
This is now ready for review.

**Review by marstamm — APPROVED** [review]
I like the step-by-step guidance we achieve with the detailed error messages 🚀

**Pull request merged**